Modules/ThirdParty/DICOMParser/src/DICOMParser/DICOMParser.cxx

```diff
@@ -137,6 +133,15 @@ bool DICOMParser::OpenFile(const dicom_stl::string& filename)
   return val;
 }
 
+void DICOMParser::CloseFile()
```

Comment:
Why not call `this->CloseFile()` at line 151? That would allow some data duplication. Are there other instances where that could be done?

Reply:
> Why not call this->CloseFile() at line 151? That would allow some data duplication.

Thanks for asking&mdash;I prefer not to call member functions in the destructor.

> Are there other instances where that could be done?

In the folder DICOMParser I checked all *.h and *.cxx files and did not find another `delete this->DataFile;`.